app/screens/edit_post/edit_post.js

```diff
@@ -220,7 +250,8 @@ export default class EditPost extends PureComponent {
                             underlineColorAndroid='transparent'
                             disableFullscreenUI={true}
                             keyboardAppearance={getKeyboardAppearanceFromTheme(this.props.theme)}
-                            onSelectionChange={this.onPostSelectionChange}
+                            onSelectionChange={(event) => this.onPostSelectionChange(event, false)}
```

Comment:
This arrow function will create a new function object every time, and since it's a prop, will trigger a re-render every time. Should move to its own function definition.

Reply:
Anonymous functions. Got it!